libraries/nprlib/nprlib_defs.mtlx

```diff
@@ -39,4 +39,18 @@
     <output name="out" type="float" default="0.0" />
   </nodedef>
 
+  <!--
+    Node: <gooch>
+    Compute Gooch shading.
+  -->
+  <nodedef name="ND_gooch" node="gooch" doc="Compute Gooch shading">
```

Comment:
We should assign this new node to the `npr` node group, so that application interfaces display it in this folder.